apartmenttherapy/oxygen-lib-ui
# Henry/transform gif

#### What does this PR do & why?
Adds a helper to check file (image) paths for a given file extension. Uses that helper to test if a url passed to `Image` is for a gif. If the image is a gif, we use a Cloudinary parameter to transform the gif into an mp4, and then we put it into a <video> tag instead of <img>.

#### How can this be manually tested?
Pull and look at `Image` use cases, primarily images in posts, and images in post galleries. 
`/new-titles-for-everyone-227653?preview=true`
`/snack-cake-recipe-264036`

The animated gifs should be replaced with mp4s

Technically `Image` is used on cover images, sponsor images (like Oxo on the `/recipes` page), and PageHeaders, but none are in use right now.

#### Any questions or concerns about this change?
Just to make sure the video is styled enough (and not in the wrong place with regard to the placeholders or not taking up the proper width)

#### Deployment Notes
There's a corresponding oxygen branch that has more styling, but no functionality additions.

#### Link to story card and other documentation
https://app.asana.com/0/543787180217047/948788415866727/f

#### GIF
![45afc41355c14f9ba5bbd9bf84731fe5b86bacbe](https://user-images.githubusercontent.com/5115651/50178172-cbac5500-02d1-11e9-8855-30447ed02a87.gif)


#### Have you remembered to...?
